reliabot/reliabot.py

```diff
@@ -138,17 +138,16 @@ def dedup_warn(message: str, key: Optional[str] = None) -> None:
 DOCTEST_OPT = "--self-test"  # Command line option for running doctests.
 DOCTEST_OPTION_FLAGS = 0  # This is modified in the re.error exception handler.
 
-RUAMEL_YAML_NOT_FOUND = True
 RUAMEL_YAML_NOT_FOUND_ERROR_MESSAGE = """
 
     Reliabot requires the ruamel.yaml module to preserve comments in dependabot.yml files.
     See https://github.com/dupuy/reliabot/#installation for installation instructions."""
 try:
     # ruamel.yaml preserves comments, PyYAML doesn't.
     from ruamel.yaml import YAML
-
 except ModuleNotFoundError as module_not_found:
     error(f"{module_not_found} {RUAMEL_YAML_NOT_FOUND_ERROR_MESSAGE}")
+# noreorder
```

Comment:
The missing hyphen seems to be necessary...

```suggestion
# no-reorder
```

Reply:
Just caught this comment, will wait for ci to finish before adding another job to the queue